.devcontainer/init_project.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -ex
+
+# Wait for docker to be ready.
+wait_for_docker() {
+  while true; do
+    docker ps > /dev/null 2>&1 && break
+    sleep 1
+  done
+  echo "Docker is ready."
+}
+
+wait_for_docker
+
+[[ -n $LD_PROJECT_TYPE ]] || LD_PROJECT_TYPE="base"
+source ./env/${LD_PROJECT_TYPE}.env || echo ""
+ddev config --web-environment-add="FRONTEND_REPOSITORY=${FRONTEND_REPOSITORY:-https://github.com/drunomics/lupus-decoupled-nuxt3-demo}"
+ddev config --web-environment-add="FRONTEND_BRANCH=${FRONTEND_BRANCH:-main}"
+# Set environment variables for both Nuxt and Next frontends.
+ddev config --web-environment-add="NUXT_PUBLIC_DRUPAL_CE_DRUPAL_BASE_URL=https://${CODESPACE_NAME}-3000.app.github.dev"
```

Comment:
CODESPACE_NAME is set by gh, right? `NUXT_PUBLIC_DRUPAL_CE_DRUPAL_BASE_URL` this looks to me like it is pointing to FE, but the variable name suggests this should be backend. Please check again. This variable name seems confusing as well.

Reply:
Good catch, should be pointing to 8080 indeed

Reply:
```suggestion
ddev config --web-environment-add="NUXT_PUBLIC_DRUPAL_CE_DRUPAL_BASE_URL=https://${CODESPACE_NAME}-8080.app.github.dev"
```

but please test it first